RELEASE-NOTES.txt

```diff
@@ -4,6 +4,7 @@
 -----
 - [**] Order Details: All unpaid orders have a Collect Payment button, which shows a payment method selection screen. Choices are Cash, Card, and Payment Link. [https://github.com/woocommerce/woocommerce-ios/pull/7111]
 - [*] Coupons: Removed the redundant animation when reloading the coupon list. [https://github.com/woocommerce/woocommerce-ios/pull/7137]
+- [internal] Login: Added "What is WordPress.com?" link in "Continue With WordPress.com" flow. [https://github.com/woocommerce/woocommerce-ios/pull/7213]
```

Comment:
I think this feature is user-facing? if we plan to run an AB test on this, we can add a note about it here

Reply:
Thank you! I made this user-facing.